.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+X. Procedure Change (2007) - Inverness (EGPE) inbound and outbound agreement amendments - thanks to @hsugden (Harry Sugden)
```

Comment:
Please in future put this below the "#Changes from"  part - if you put it on top it will not spot there is a merge conflict and just add your comment to the top, so i have to edit every one manually

Reply:
Changed in all my other PRs